### PR TITLE
PUB-2306 Family and mixed lists - Move party to case level

### PR DESCRIPTION
**Pull request by KianKwa**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-2306

### Change description ###

Family and mixed lists - Move party to case level

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/service/listManipulation/CivilFamilyAndMixedListService.ts [hunk 1]
```diff
@@ -15,11 +15,39 @@ export class CivilFamilyAndMixedListService {
      * Manipulate the family/mixed json data for rendering to the screen.
      * @param list
      */
-    public sculptedFamilyMixedListData(list: string): object {
+    public sculptedFamilyMixedListData(list: string, partyAtHearingLevel = false): object {
```
Comment (ChrisS1512) [inline src/main/service/listManipulation/CivilFamilyAndMixedListService.ts:18]: I'm wondering if this method adds value. A similar if statement is done in the controller. Could the controller just call the sculpted list data methods this one calls directly?

File: src/main/service/listManipulation/CivilFamilyAndMixedListService.ts [hunk 2]
```diff
@@ -15,11 +15,39 @@ export class CivilFamilyAndMixedListService {
...
+        if (partyAtHearingLevel) {
+            return this.sculptedListDataV1(list, true);
+        }
         return this.sculptedListData(list, true);
     }
 
     private sculptedListData(list: string, isFamilyMixedList = false): object {
```
Comment (ChrisS1512) [inline src/main/service/listManipulation/CivilFamilyAndMixedListService.ts:25]: I think this method / equivilant V1 below would be better replaced with something like sculptedListDataHearingLevel etc, just to make it clear. What do you think?

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/service/listManipulation/CivilFamilyAndMixedListService.ts (hunk 2 above)
Comment (KianKwa) [reply src/main/service/listManipulation/CivilFamilyAndMixedListService.ts]: Renamed to sculptedListDataPartyAtHearingLevel()

File: src/main/service/listManipulation/CivilFamilyAndMixedListService.ts (hunk 1 above)
Comment (KianKwa) [reply src/main/service/listManipulation/CivilFamilyAndMixedListService.ts]: Have removed the top level methods so the controller calls them directly.

**Review by ChrisS1512 — DISMISSED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**